### PR TITLE
add configuration.md

### PR DESCRIPTION
**Pull request by yuedongli1**
Thank you for your contribution to the MindYOLO repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindyolo/blob/master/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: tutorials/configuration_CN.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+## 基础参数
+
+### 参数说明
+   - device_target: 所用设备，Ascend/GPU/CPU
+   - save_dir: 运行结果保存路径，默认为./runs
+   - log_interval: 打印日志step间隔，默认为100
+   - is_parallel: 是否分布式训练，默认为False
+   - ms_mode: 使用静态图模式(0)或动态图模式(1)，默认为0。
+   - config: yaml配置文件路径
+   - per_batch_size: 单卡batch size，默认为32
```
Comment (zhanghuiyao) [inline tutorials/configuration_CN.md:18]: 单卡 -> 每张卡

File: tutorials/configuration_CN.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+## 数据增强
+
+### 参数说明
+   - num_parallel_workers: 读取数据的工作进程数
+   - train_transformers: 训练过程数据增强
+   - test_transformers: 验证过程数据增强
+   - ......
+
+### yaml文件样例
+该部分参数在[configs/yolov3/hyp.scratch.yaml](../configs/yolov3/hyp.scratch.yaml)中定义，其中train_transformers和test_transformers均为由字典组成的列表，各字典内部为各数据增强的相关参数
```
Comment (zhanghuiyao) [inline tutorials/configuration_CN.md:74]: 各字典内部为各数据增强的相关参数
->
字典包含，数据增强操作名称、发生概率及该增强方法相关的参数

File: tutorials/configuration_CN.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+- 用户命令行传入参数
+- python执行py文件中parser的默认参数
+- 命令行传入config参数对应的yaml文件参数
+- 命令行传入config参数对应的yaml文件中__BASE__参数中包含的yaml文件参数，例如yolov3.yaml含有如下参数：
+```yaml
+__BASE__: [
+  '../coco.yaml',
+  './hyp.scratch.yaml',
+]
+```
```
Comment (zhanghuiyao) [inline tutorials/configuration_CN.md:246]: 参数继承关系可以放到上面说

File: tutorials/configuration_CN.md [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+
+### 参数说明
+
+   - model_name: 模型名称
+   - depth_multiple: 模型深度因子
+   - width_multiple: 模型宽度因子
+   - stride: 特征图下采样倍数
+   - anchors: 预设锚框
+   - backbone: 模型骨干网络
+   - head: 模型检测头
```
Comment (zhanghuiyao) [inline tutorials/configuration_CN.md:109]: - ...
    - 注意：yolov3会根据backbon和head参数进行网络构建，参数以列表形式呈现，包含4个参数，分别是 输入层编号(-1代表上一层)、层数、模块名称和模块相应参数；用户也可以不通过backbone和head提前定义和注册相应网络；

File: tutorials/configuration_CN.md [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+           'hair drier', 'toothbrush' ]
+ ```
+
+## 数据增强
+
+### 参数说明
+   - num_parallel_workers: 读取数据的工作进程数
+   - train_transformers: 训练过程数据增强
+   - test_transformers: 验证过程数据增强
+   - ......
```
Comment (zhanghuiyao) [inline tutorials/configuration_CN.md:71]: ...... -> ...

**Review by CaitinZhao — APPROVED** [review]

**Review by zhanghuiyao — APPROVED** [review]

**Pull request merged**